shared/components/LoginFacebook.ios.js

```diff
@@ -1,59 +1,67 @@
 import React, { Component, PropTypes } from 'react';
 import {
-    Text,
-    View,
-    TouchableHighlight
+  Text,
+  View,
+  TouchableOpacity,
+  Dimensions,
+  Platform
 } from 'react-native';
 import {FBLoginManager} from 'react-native-facebook-login';
+import { Actions } from 'react-native-router-flux';
+import {
+  setLoginType,
+  grantFacebookUser,
+  removeUserToken,
+  removeLoginType
+} from '../actions/authActions';
+
+const WindowHeight = 477.8 + 162;
+const WindowWidth = 360;
```

Comment:
no it is not. They are used as a reference to the ratio we have in Zeplin destine